udata/templates/home.html

```diff
@@ -42,107 +36,44 @@ <h1 class="page-header">{{last_post.name}}
 
             </div>
         </div>
-        {% endif %}
-
-        <div class="item">
-            <div class="container">
 
-                <div class="row">
```

Comment:
These removals seem to be a major home page change. Can we have some screenshots showing before/after this change ?